tests/test_handlers.py

```diff
@@ -636,6 +636,21 @@ def test_am1elf10_conformer_independence():
         assert np.sum(delta_charges) == pytest.approx(0.0)
 
 
+def test_trans_carboxlic_acid():
+    # Test fallback to turn off hydrogen sampling if charge generation failed
+    # due to trans-COOH
+    with resources.path("timemachine.testsystems.data", "mobley_820789.sdf") as path_to_ligand:
+        suppl = Chem.SDMolSupplier(str(path_to_ligand), removeHs=False)
+    mol = next(suppl)
+    rdmolops.AssignStereochemistryFrom3D(mol, confId=0, replaceExistingTags=True)
+    am1elf10_charges = nonbonded.oe_assign_charges(mol, charge_model=nonbonded.AM1ELF10)
+
+    AllChem.EmbedMolecule(mol, randomSeed=2022, useRandomCoords=True)
+    new_am1elf10_charges = nonbonded.oe_assign_charges(mol, charge_model=nonbonded.AM1ELF10)
+    delta_charges = np.abs(np.array(am1elf10_charges) - np.array(new_am1elf10_charges))
```

Comment:
nit: np.array conversions may be unneeded, since oe_assign_charges already returns an array https://github.com/proteneer/timemachine/blob/5c45ad0539fb20d9d111bc6f3ecfb46f636e3ba4/timemachine/ff/handlers/nonbonded.py#L111